app/src/features/Mappings/__tests__/CreateMapping.test.tsx

```diff
@@ -41,6 +42,16 @@ const handlers = [
         ctx.json<ApiOwnersListApiResponse>([owner])
       )
   ),
+  rest.get(
+    new RegExp("http://example.com/api/fhir/ValueSet/resource-types/\\$expand"),
```

Comment:
why using a regex here ?

Reply:
This is the best solution we found with @tevariou for escaping the `$` char as it is not correctly parsed by MSW